README.md

```diff
@@ -1,6 +1,6 @@
 # Cargo B(inary)Install
 
-`cargo binstall` provides a low-complexity mechanism for installing rust binaries as an alternative to building from source (via `cargo install`) or manually downloading packages. This is intended to work with existing CI artifacts and infrastructure, and with minimal overhead for package maintainers.
+`cargo binstall` provides a low-complexity mechanism for installing rust binaries as an alternative to building from a source (via `cargo install`) or manually downloading packages. This is intended to work with existing CI artifacts and infrastructure, and with minimal overhead for package maintainers.
```

Comment:
"building from source" is ok here.
"building from a source" actually sounds confusing.